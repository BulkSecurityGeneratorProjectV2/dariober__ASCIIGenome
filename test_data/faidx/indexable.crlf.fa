>seq1
NNNNNNNNNNNNNNNNNNNNNNN
NNNNNNNNN
>seq2
NNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNNNNNN
NNNNNNNNNNNNNNN

