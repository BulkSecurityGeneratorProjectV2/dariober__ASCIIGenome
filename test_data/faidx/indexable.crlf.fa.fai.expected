seq1	32	7	23	25
seq2	53	50	19	21
